components/playground/instance/tiflash.go

```diff
@@ -26,7 +26,7 @@ import (
 	"time"
 
 	"github.com/pingcap/errors"
-	"github.com/pingcap/tiup/components/playground/api"
+	"github.com/pingcap/tiup/pkg/cluster/api"
```

Comment:
So "playground" is somehow relied on "cluster"?

Reply:
the api of PDClient is copied from cluster. I think I will move them to a common package afterwards.

Reply:
@JaySon-Huang It actually relies on `pkg`, `cluster` is a sub-directory of `pkg`, different from module `cluster`